docs/source/datasources/usgs_ndvi.rst

```diff
@@ -1,5 +1,5 @@
 USGS
```

Comment:
What I meant with my comment is if you could change this title to NDVI instead of USGS -- that way "NDVI" would show up in the list of data sets instead of USGS (so a bit more descriptive for people who are trying to see what's available). But no worries if you disagree and we can keep discussing in the relevant issues

Reply:
No that totally makes sense, I didn't understand exactly what you meant!

Reply:
I forgot to change in here but have put in the PR directly about making clear the issues with USGS NDVI data.